openassessment/management/commands/create_oa_submissions_from_file.py

```diff
@@ -0,0 +1,363 @@
+
+        missing_usernames = set(usernames) - set(anonymous_user_ids.keys())
+        if missing_usernames:
+            raise CommandError("Unable to load anonymous id for user(s) " + ' '.join(missing_usernames))
+
+        self.username_to_anonymous_user_id = anonymous_user_ids
+
+    def get_display_names(self, submissions_config):
+        """
+        Returns a list of all ORA display names mentioned in the submissions config
+        """
+        display_names = []
+        for ora_config in submissions_config:
+            display_name = ora_config['displayName']
+            if display_name in display_names:
+                raise CommandError('Duplicate ORA display name found:' + display_name)
+            display_names.append(display_name)
+        return display_names
+
+    def load_ora_blocks(self, course_id, display_names):
+        """
+        Look up openassessment blocks for the course from the modulestore, and save the ones that match the given
+        display names in a dict mapping from display name to block in self.display_name_to_block
+        """
+        try:
+            from xmodule.modulestore.django import modulestore  # pylint: disable=import-error
+        except ModuleNotFoundError as e:
+            raise CommandError((
+                "Cannot import xmodule.modulestore.django.modulestore. "
+                "This management command must be run from the LMS shell."
+            )) from e
+        openassessment_blocks = modulestore().get_items(
+            course_id, qualifiers={'category': 'openassessment'}
+        )
+        display_name_to_block = {}
+        for block in openassessment_blocks:
+            if block.parent is not None:
+                display_name_to_block[block.display_name] = block
+
+        missing_display_names = set(display_names) - set(display_name_to_block.keys())
+        if missing_display_names:
+            raise CommandError(
+                f"The following Display Name(s) were not found in {str(course_id)} {' '.join(missing_display_names)}"
+            )
+        self.display_name_to_block = display_name_to_block
+
+    def init_ora_test_data(self, course_id, submissions_config):
+        """
+        Run the initialization. Create all users, enroll in the course, and then call the submit logic.
+        """
+        learners, course_staff = self.get_usernames(submissions_config)
+
+        print(f'Creating and enrolling {len(learners)} learners')
+        call_command(
+            'create_test_users',
+            *learners,
+            course=course_id,
+            ignore_user_already_exists=True
+        )
+
+        print(f'Creating and enrolling {len(course_staff)} course staff')
+        call_command(
+            'create_test_users',
+            *course_staff,
+            course=course_id,
+            course_staff=True,
+            ignore_user_already_exists=True
```

Comment:
This is a new flag in that command, wil link my PR as soon as it's up

Reply:
https://github.com/edx/edx-platform/pull/29746

Reply:
If you want to remove this line for testing, that's fine, but just know that every time you run it, all usernames mentioned in the doc will need to be globally unique.